forte/ontology_specs/medical.json

```diff
@@ -4,11 +4,11 @@
     "base_ontology.json"
   ],
   "additional_prefixes": [
-    "ftx.medical"
+    "ftx.medical.clinical"
```

Comment:
so we would probably want to make sure we don't have the same module defined here and forte-medical. Since the code here is for demo/example purposes, we may use a package path like `ftx.demo.clinical` or `ftx.onto.clinical`.